### ISSUE TITLE
simplejson==3.18.1 not supported

### ISSUE THREAD

**Issue by marcin2x4** [aws/aws-lambda-python-runtime-interface-client#93]
When installing `office365` package in lambda's `/tmp/ dir:

```
os.system('/var/lang/bin/pip3.9 install office365 -t /tmp/')
import office365

def handler(event, context): 
    print(sys.path)
    print("hi!")
    print(office365.__version__)
```

one of the dependencies threw below error:
```
awslambdaric 2.0.4 requires simplejson==3.17.2, but you have simplejson 3.18.1 which is incompatible.
Module office365 not found
```

**Comment by kpark-hrp** [comment]
@alex-pewpew @briensea Can we please upgrade out of this ancient `simplejson` version? Or at least loosen the pin

**Comment by kpark-hrp** [comment]
Should be at least minimum `simplejson>=3.18.1`

**Comment by briensea** [comment]
Fixed in [2.0.11](https://github.com/aws/aws-lambda-python-runtime-interface-client/releases/tag/2.0.11).
